src/main/java/github/scarsz/discordsrv/objects/threads/ConsoleMessageQueueWorker.java

```diff
@@ -19,13 +19,25 @@
 package github.scarsz.discordsrv.objects.threads;
 
 import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.objects.ConsoleMessage;
 import github.scarsz.discordsrv.util.DiscordUtil;
+import github.scarsz.discordsrv.util.LangUtil;
 import net.dv8tion.jda.api.JDA;
 import net.dv8tion.jda.api.entities.Message;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.WordUtils;
+
+import java.util.Deque;
 
 public class ConsoleMessageQueueWorker extends Thread {
 
+    private static final char LINE_WRAP_INDENT = '\t';
+    private static final int MIN_SLEEP_TIME_MILLIS = 1000;
```

Comment:
This should be higher that the *absolute minimum allowed by Discord*

Reply:
This was preexisting behavior, my only change here is moving the existing `1000` magic number to a constant. 

JDA should be handling the bulk of actual API rate limiting, assuming it is correctly configured.

What value would you prefer this be changed to?

Reply:
Yes JDA would handle it, but if it would hit a rate limit even once, it would then have one message always waiting in ratelimit limbo to be sent (assuming there's enough things to send every second), I'd suggest 1500 or 2000

Reply:
Just to be clear, this is a behavioral change to what currently exists in the master/develop branches:
https://github.com/DiscordSRV/DiscordSRV/blob/develop/src/main/java/github/scarsz/discordsrv/objects/threads/ConsoleMessageQueueWorker.java#L61

From what I understand, this is a safety mechanism to prevent the user from setting the value too low in `DiscordConsoleChannelLogRefreshRateInSeconds`. 

Has 1000 ms been found too low? Is there an associated bug with the existing behavior? Of course, I can make this change, I just want to understand the motivation for it and why it has not yet been addressed in the existing implementation.

Reply:
As an example https://github.com/DiscordSRV/DiscordSRV/issues/719 Discord changed the channel update ratelimit to 5 minutes, back then DiscordSRV had a minimum of 5 minutes on the topic updater, yet it still didn't behave properly because it would be a few milliseconds wrong. This hasn't been an issue, atleast that I know of, since the default value fir this is 5 seconds in the config and *most* people don't have flooding console output. We tend not to go with the bare minimum in terms of rate limits, but I just hadn't spotted this until now

Reply:
No problem. I really appreciate the explanation. I will change it to 2000, since coincidentally that is what I use in my own configuration.